### ISSUE TITLE
Request: add latest Docker Compose in the next release

### ISSUE THREAD

**Issue by mathiasimmer** [hypriot/image-builder-rpi#161]
Current version of Docker Compose included in the image does not support the new Docker Compose file formats (3.0 and 3.1) so I would like to make a request that Docker Compose is updated to the latest release ( 1.11.2  as of now) which supports the new formats.

**Comment by StefanScherer** [comment]
Thanks @mathiasimmer 
We also considered updating compose, but the build of a docker-compose binary failed and it's hard to debug.

As a workaround just do the following to get the latest docker-compose

```bash
sudo apt-get remove docker-compose
sudo apt-get install python-pip
pip install docker-compose
```

**Comment by StefanScherer** [comment]
Fixed with release 1.4.0
